### PR TITLE
[#725]:Fixed Badges Static Errors(connects #725)

### PR DESCRIPTION
**Pull request by Pr0chin**

### CODE REVIEW

**Comment by Pr0chin** [comment]
badges

**Review by dogi — APPROVED** [review]

**Comment by dogi** [comment]
@Pr0chin @razu9861 nice picture ;)

File: app/MyApp/index.html [hunk 1]
```diff
@@ -1017,10 +1017,13 @@
             <%= stepNo %>
         </td>
         <td>
-            <%if(status == "1"  ){%>
-                <%=App.languageDict.attributes.Submitted%>
-            <%}else{%>
-                <%=App.languageDict.attributes.Un_Submitted%>
+            <%for (var i=1; i<status.length; i++) {%>
```
Comment (lmmrssa) [inline app/MyApp/index.html]: Status is for result 'Submitted/Unsubmitted' will be based upon does status for last attempt exist or not. If exist and status is -1 then also its unsubmitted

**Review by lmmrssa — CHANGES_REQUESTED** [review]

**Review by lmmrssa — APPROVED** [review]

**Review by nirojdyola — APPROVED** [review]

**Comment by dogi** [comment]
@lmmrssa good catch ;)

**Review by leomaxi — APPROVED** [review]

**Pull request closed without merging**